src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
```

Comment:
The `health` parameter in the `Animal` constructor is not used. You are setting `this.health` to 100 regardless of the `health` argument passed. If you intend to use the `health` parameter, you should assign it to `this.health`.